Firestore/Example/Tests/API/FIRAggregateQueryUnitTests.mm

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <FirebaseFirestore/FIRAggregateQuery.h>
+
+#import <FirebaseFirestore/FIRQuery.h>
+
+#import <XCTest/XCTest.h>
+
+#import "Firestore/Source/API/FIRFirestore+Internal.h"
+#import "Firestore/Source/API/FIRQuery+Internal.h"
+
+#import "Firestore/Example/Tests/API/FSTAPIHelpers.h"
+#import "Firestore/Example/Tests/Util/FSTHelpers.h"
+
+#include "Firestore/core/src/core/query.h"
+#include "Firestore/core/test/unit/testutil/testutil.h"
+
+namespace api = firebase::firestore::api;
+using firebase::firestore::testutil::Query;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRAggregateQueryUnitTests : XCTestCase
+@end
+
+@implementation FIRAggregateQueryUnitTests
+
+- (void)testEquals {
```

Comment:
This test method is called "testEquals" but it appears to be testing the _hash_ method. This seems inconsistent to me. Am I missing something?

Reply:
Ahh, yes that other test also suffers from the not-quite-correct name. Your name suggestions LGTM.

Reply:
This seems more pervasive. Can we leave this as is for now.

<img width="1284" alt="image" src="https://user-images.githubusercontent.com/13738054/236841898-ab8de7a1-91b9-4f18-ae0a-2e6fc062829f.png">